install/helm/agones/templates/extensions-deployment.yaml

```diff
@@ -123,6 +123,10 @@ spec:
           httpGet:
             path: /live
             port: http
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: 8080
           initialDelaySeconds: {{ .Values.agones.extensions.healthCheck.initialDelaySeconds }}
```

Comment:
the way you've added this, the `initialDelaySeconds` etc. are applied to the `readinessProbe` but were previously configured on the liveness probe. I would add a `.readiness` with similar settings (it looks like we default to 3/3/3 in values.yaml, that SGTM for readiness as well).

Reply:
I see, created a section for readinessProbe and added a section in .Values as well for readiness.